hack/test-integration.sh

```diff
@@ -137,12 +134,14 @@ export childargs
 # hack/test-integration.sh WatchBuilds
 # hack/test-integration.sh Template*
 # hack/test-integration.sh "(WatchBuilds|Template)"
-
+tests=( $(go run "${OS_ROOT}/hack/listtests.go" -prefix="${OS_GO_PACKAGE}/${package}.Test" "${testdir}" | grep -E "${1-Test}") )
```

Comment:
this fails if you specify a pattern that doesn't match any tests... I guess that's ok